guides/README.md

```diff
@@ -95,6 +95,20 @@ All PDFs should be available for download though.
 
 Please read these guidelines before opening a Pull Request. For more information, see [Guidelines for Red Hat Documentation](https://redhat-documentation.github.io/).
 
+### Creating new guide
+
+Each guide must be in a separate directory in guides/ prefixed with "doc-". The following requirements must be met:
+
+* Top-level file `master.adoc`.
+* Directory `topics` with actual content.
+* Symlink `common` to `../common` for shared content.
+* Directory `images` with images.
+* Symlink `images/common` to `../common/images`.
+* File `Makefile` which includes `../common/Makefile`.
+* File `docinfo.xml` for downstream (Red Hat) guides. Presence of this file will cause build check via `ccutil`, a tool used by Red Hat documentation team.
+
```

Comment:
I would appreciate a recommendation to automatically put new content into the `guides/common/modules` directory and stick to one (level one) heading per file.

Everything comes together in either the master.adoc or assemblies, which are lists of files with optional leveloffsets.

Reply:
Done.